examples/loaders/gtfs_loader.ipynb

```diff
@@ -0,0 +1,132 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# GTFS Loader Example"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pathlib import Path\n",
+    "from srai.loaders import GTFSLoader\n",
+    "import gtfs_kit as gk\n",
+    "import geopandas as gpd\n",
+    "import numpy as np\n",
+    "from shapely.geometry import Point\n",
+    "from srai.utils.constants import WGS84_CRS"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Download an example GTFS feed from Wroclaw, Poland\n",
+    "\n",
+    "In this notebook we use the GTFS feed for Wroclaw, Poland as an example, which is available in Wroclaw's open data repository[1]. This download uses transitfeeds.com[2] to download the feed, but you can also download the feed directly from the Wroclaw open data repository.\n",
+    "\n",
+    "1. https://www.wroclaw.pl/open-data/dataset/rozkladjazdytransportupublicznegoplik_data\n",
+    "2. https://transitfeeds.com/p/mpk-wroc-aw/663"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "! wget -O files/example.zip https://transitfeeds.com/p/mpk-wroc-aw/663/20221221/download"
```

Comment:
We should replace this `wget` dependency and implement a dedicated download function.
https://gist.github.com/yanqd0/c13ed29e29432e3cf3e7c38467f42f51
https://stackoverflow.com/questions/37573483/progress-bar-while-download-file-over-http-with-requests

Reply:
Done ;)